onnxruntime/test/onnx/main.cc

```diff
@@ -311,13 +311,8 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
       {"BatchNorm2d_momentum_eval", "disable reason"},
       {"BatchNorm3d_eval", "disable reason"},
       {"BatchNorm3d_momentum_eval", "disable reason"},
-#if defined(__GNUG__) && !defined(__LP64__)
```

Comment:
Pls make sure that x86 linux pipeline is fine with this.

Reply:
Yes, this is something I am watching and will address should the issue arise.

---
In reply to: [291766216](https://github.com/microsoft/onnxruntime/pull/1059#discussion_r291766216) [](ancestors = 291766216)